devicon.json

```
@@ -3623,35 +3653,33 @@
           }
         ]
     },
-    {
-        "name": "spring",
+	{
+        "name": "subversion",
         "tags": [
-            "framework"
+            "svn",
+            "version"
         ],
         "versions": {
             "svg": [
                 "original",
-                "original-wordmark",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original",
+                "original-wordmark"
```

My bad. The `original-wordmark` doesn't fit being an icon.
```suggestion
                "original"
```